### PR TITLE
Use setuptools infrastructure for testing; show 'install dictorm[postgresql]'

### PR DESCRIPTION
**Pull request by jean**
Add 'green' as alias for 'test' (see https://github.com/CleanCut/green#setuppy-command).

With this PR I'm trying to get the most out of what setuptools gives us: 

- make plain `python setup.py test` work (install  `tests_require`, run tests);
- use the syntax for installing extras (`dictorm[postgresql]`).

### CODE REVIEW

**Comment by jean** [comment]
I don't think this is due to these changes (`NameError: name 'DictCursor' is not defined`, [ref](https://travis-ci.org/rolobio/DictORM/jobs/309360509#L604)).
Tests pass locally:

```
(DictORM) 13:42 john@john:~/git/DictORM(master)$ python  setup.py test
running green
Searching for coveralls
Best match: coveralls 1.2.0
Processing coveralls-1.2.0-py3.5.egg

Using /home/john/repos/git/DictORM/.eggs/coveralls-1.2.0-py3.5.egg
Searching for coverage
Best match: coverage 4.4.2
Processing coverage-4.4.2-py3.5.egg

Using /home/john/repos/git/DictORM/.eggs/coverage-4.4.2-py3.5.egg
Searching for docopt>=0.6.1
Best match: docopt 0.6.2
Processing docopt-0.6.2-py3.5.egg

Using /home/john/repos/git/DictORM/.eggs/docopt-0.6.2-py3.5.egg
.............................x........................................x................................

Ran 103 tests in 7.040s

OK (expected_failures=2, passes=101)
```

**Comment by jean** [comment]
I have one remaining question: https://github.com/CleanCut/green/issues/174

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,12 @@ python:
     - "3.4"
     - "3.5"
 install:
-    - "pip install -r requirements.testing.txt"
+    - "pip install -e .[postgresql]"
```
Comment (rolobio) [inline .travis.yml:7]: I was looking for how to do that! Great change!

**Review by rolobio — APPROVED** [review]
Fantastic changes.  I really like that setup.py now handles installing/running testing.  Also, good catch with the "sudo" command for docker.

**Pull request merged**